### DISCUSSION TITLE
Temporarily disable 'compass' icon that flies to your current location

### DISCUSSION THREAD

**Discussion by bwaslo** [organicmaps/organicmaps#2430] [Ideas]
Usung the app to plan out an overseas city walking trip.  Keep accidentally touching this ##$_& icon (at lower right on map), which zooms me back thousands of miles to home.  Much cursing ensues.  Wiuld be great if that icon could be temporarily disabled when using app for this!

**Comment by biodranik** [comment]
Is it iOS or Android? Are you speaking about the "my position" icon, that centers on your position? You can tap anywhere on the empty map to hide all buttons completely.

**Comment by bwaslo** [comment]
Thanks for replying -- It's an Android.  The little icon is there
after searching for a site (such as "Barcelona, Spain", looks like a black
circle with 4 spikes around it and a dot in the middle -- when it gets
(accidentally, always) touched, it becomes a blue arrow pointing to about 2
o'clock position and sends the position sailing back to where I am in the
Pacific Northwest USA.  I don't understand what is meant by "empty map", it
always has something on it!.

**Reply by biodranik** [reply]
Can you please record a short video of how exactly you accidentally touched the button? Nobody complained so far about it :)

Find any gray area (empty map) on the screen and touch it. Buttons should disappear.
